### PR TITLE
Admin adds item

### PR DESCRIPTION
**Pull request by mrjaimisra**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -9,11 +9,18 @@
...
-  # namespace :admin do
+  # namespace "admin/dashboards" do
   #   resources :users, only: [:index]
   # end
 
-  get "/admin/dashboard", to: "admin/dashboard#index"
+  namespace "admin" do
+    resources :items, only: [:new, :create, :edit, :destroy]
+
+    # get 'dashboards', to: "dashboards#show"
```
Comment (applegrain) [inline config/routes.rb]: Let's remove this comment.

File: spec/features/admin_add_item_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+
+RSpec.describe "an admin on their dashboards" do
+  it "can access the form to add a new item" do
+    register_admin
+    sign_in
+
+    visit "/admin/dashboard"
+    click_link "Add New Meal"
+
```
Comment (applegrain) [inline spec/features/admin_add_item_spec.rb]: Can you scope these tests within a css-selector?

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -21,7 +21,7 @@
       expect(user).to_not be_valid
     end
 
-    it "has a default role of 'default'" do
+    xit "has a default role of 'default'" do
```
Comment (applegrain) [inline spec/models/user_spec.rb]: do we want this skipped?

File: spec/features/admin_add_item_spec.rb (hunk 2 above)
Comment (mrjaimisra) [inline spec/features/admin_add_item_spec.rb]: yes, I'll bootstrap them now.

File: config/routes.rb (hunk 1 above)
Comment (mrjaimisra) [inline config/routes.rb]: removed that and other irrelevant stuff from the routes

File: spec/models/user_spec.rb (hunk 3 above)
Comment (mrjaimisra) [inline spec/models/user_spec.rb]: unskipped in later commit

File: config/routes.rb [hunk 4]
```diff
@@ -10,11 +10,10 @@
   post "/login", to: "sessions#create"
   delete "/logout", to: "sessions#destroy"
 
-  # namespace :admin do
-  #   resources :users, only: [:index]
-  # end
-
-  get "/admin/dashboard", to: "admin/dashboard#index"
+  namespace "admin" do
```
Comment (applegrain) [inline config/routes.rb]: nice namespacing!

**Comment by noobjey** [comment]
nice

**Pull request merged**